README.rst

```diff
@@ -365,6 +365,10 @@ Building DALI using Clang (experimental):
 -  ``BUILD_NVDEC`` - build with ``NVIDIA NVDEC`` support (default: ON)
 -  ``BUILD_NVML`` - build with ``NVIDIA Management Library`` (``NVML``) support (default: ON)
 -  ``WERROR`` - treat all build warnings as errors (default: OFF)
+-  ``BUILD_WITH_ASAN`` - build with ASAN support (default: OFF). To run issue: LD_LIBRARY_PATH=.
+   ASAN_OPTIONS=symbolize=1:protect_shadow_gap=0 ASAN_SYMBOLIZER_PATH=$(shell which llvm-symbolizer)
+   LD_PRELOAD=*PATH_TO_LIB_ASAN*/libasan.so.*X* *PATH_TO_BINARY*
+   where *X* depends on used compiler version (for example GCC 7.x uses 4)
```

Comment:
I would add that it probably won't work with an earlier one.

Reply:
Done